test/functional/feature_pruning.py

```diff
@@ -41,28 +41,14 @@ def mine_large_blocks(node, n):
         mine_large_blocks.nTime = 0
 
     # Get the block parameters for the first block
-    big_script = CScript([OP_RETURN] + [OP_NOP] * 950000)
     best_block = node.getblock(node.getbestblockhash())
     height = int(best_block["height"]) + 1
     mine_large_blocks.nTime = max(mine_large_blocks.nTime, int(best_block["time"])) + 1
     previousblockhash = int(best_block["hash"], 16)
+    bigScript = CScript([OP_RETURN] + [OP_TRUE] * 950000)
```

Comment:
In commit "rpc: Enable wallet import on pruned nodes and add test" (05463d4001001213c17672c1607a3a493927ddd7)

Can just keep existing `big_script` variable, no need to replace it with new `bigScript` variable. This would make the diff small and make the naming convention more convenient

Reply:
my bad, fixed